packages/round-manager/src/features/program/ViewProgramPage.tsx

```diff
@@ -302,50 +298,48 @@ export default function ViewProgram() {
                 className="object-cover pl-6 pr-4"
               />
             </button>
-            {process.env.REACT_APP_DIRECT_GRANT_ENABLED && (
-              <button
-                onClick={() => setGrantType("directGrant")}
-                className={`flex w-full rounded border  ${
+            <button
+              onClick={() => setGrantType("directGrant")}
+              className={`flex w-full rounded border  ${
+                grantType === "directGrant"
+                  ? "border-violet-400"
+                  : "border-grey-100"
+              } bg-white p-6 cursor-pointer`}
+              style={{
+                boxShadow:
                   grantType === "directGrant"
-                    ? "border-violet-400"
-                    : "border-grey-100"
-                } bg-white p-6 cursor-pointer`}
-                style={{
-                  boxShadow:
+                    ? " 0px 10px 15px -3px #0000001A"
```

Comment:
Is there any tailwind class that would allow avoiding the custom style?

Reply:
shadow-lg is the closest, i'll see if it looks the same.

Reply:
it's the same, fixed.